example/iframe.html

```diff
@@ -23,8 +23,8 @@
     }
   </script>
   <script
-    src="./dist/src/index.js"
-    onload="GENERATOR.initialize(['http://127.0.0.1:3000', 'http://localhost:3000'], onDownload)"
+    src="./dist/wallet-generator.js"
+    onload="GENERATOR.initialize(['http://127.0.0.1:3000', 'http://localhost:3000', 'https://amplicalabs.github.io'], onDownload)"
```

Comment:
I mean Wil may be changing this in less than an hour? the domain that is

Reply:
yeah i realized late but will get it running once we are up